### PR TITLE
add support for ssm maintenance window tasks

### PR DESCRIPTION
**Pull request by pinzon**
This PR adds support for some operations related to Maintenance Window Tasks:

- [X] register_task_with_maintenance_window
- [X] describe_maintenance_window_tasks
- [X] deregister_task_from_maintenance_window

I'll rebase it after #6429 is merged

### CODE REVIEW

**Comment by mikegrima** [comment]
Codecov complains that there are 2 spots not covered by tests. Mostly a nitpick, but we would prefer you have coverage for it.

File: moto/ssm/models.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import re
+import uuid
```
Comment (bblommers) [inline moto/ssm/models.py:2]: This should use the `mock_random` as well

File: tests/test_ssm/test_ssm_maintenance_windows.py [hunk 2]
```diff
@@ -274,3 +274,110 @@ def test_deregister_target_from_maintenance_window():
         WindowId=window_id,
     )
     resp.should.have.key("Targets").should.have.length_of(0)
+
+
+def test_describe_maintenance_window_with_no_task_or_targets():
```
Comment (bblommers) [inline tests/test_ssm/test_ssm_maintenance_windows.py:279]: Looks like this is missing the `mock_ssm`-decorator

**Review by bblommers — CHANGES_REQUESTED** [review]

**Review by bblommers — APPROVED** [review]
Awesome - thanks @pinzon!

**Pull request merged**